### DISCUSSION TITLE
Story of using OpenAP

### DISCUSSION THREAD

**Discussion by drudrum** [TUDelft-CNS-ATM/openap#11] [Show and tell]
I used the library to calculate emissions and climb and descent profiles. Average fuel consumption was used for the rest of the aircraft.

We counted how much fuel economy could be when flying by FRA(Free Route Airspace) at one FIR of Russia.

https://git.ans.aero/FRA/fraModel/src/master/helpDocs/screenshots.md

The idea behind the library is simple. I think this path is correct. But I want to collect the characteristics of aircraft and adjust their parameters according to statistics.

Thanks for the work you've done.

**Comment by junzis** [comment]
Nice work @drudrum! Thank you for sharing this!
